cmd/tools/rpcwrappers/main.go

```diff
@@ -105,12 +107,14 @@ var (
 		"client.history.DeleteDLQTasks":         true,
 		"client.history.ListQueues":             true,
 		"client.history.ListTasks":              true,
+		"client.history.HealthCheck":            true,
```

Comment:
why is this here? there's no call by this name in HistoryService (or MatchingService below)